From patchwork Thu Apr 22 04:29:24 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 12217623
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-10.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,
	MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS autolearn=unavailable
	autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 57E90C433B4
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:29:32 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 3C6F961222
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:29:32 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S229833AbhDVEaF (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 00:30:05 -0400
Received: from lindbergh.monkeyblade.net ([23.128.96.19]:45026 "EHLO
        lindbergh.monkeyblade.net" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S229561AbhDVEaE (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 00:30:04 -0400
Received: from ozlabs.org (bilbo.ozlabs.org [IPv6:2401:3900:2:1::2])
        by lindbergh.monkeyblade.net (Postfix) with ESMTPS id 8E726C06174A;
        Wed, 21 Apr 2021 21:29:29 -0700 (PDT)
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 4FQkw52jX2z9sRf;
        Thu, 22 Apr 2021 14:29:25 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1619065765;
        bh=S7qxoLL2mxanY8fwURHisQAFOqNZ2RlcleQrIk620V0=;
        h=Date:From:To:Cc:Subject:From;
        b=Zb6uhxXf2hONrLKwxnLkIbF2Sh2kwZ0g0r8Ao9o1gEKe2hi22rJ+UKrY7qx/+lhBS
         0SxKnMghLD48aYJwLfWYRGH2UivkhgElkf/PIusCXpLsueyOyG0jVcC1+e2jWEFUt9
         FtBDo/lSMROE6rNBk1Ttr9rioTueKzXyPhnEADnh1b/YjEirbeYCCezvYrQgP8UMVD
         xOPI3tI3yugflgV+oLf7ouegXMhY6x02gQW+/QN2G5ilMU+njl6v2BrjEiXVNdsouD
         9hVcmXD1YB6yljwJziisKZmq/PPouT7IMvB1xBFvDybQu4SUIA8//ypz/jSJA9ukpZ
         RwwwnQl9jZfcA==
Date: Thu, 22 Apr 2021 14:29:24 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>
Cc: Emanuele Giuseppe Esposito <eesposit@redhat.com>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Linux Next Mailing List <linux-next@vger.kernel.org>,
        Sean Christopherson <seanjc@google.com>
Subject: linux-next: manual merge of the kvm tree with Linus tree
Message-ID: <20210422142924.222f7216@canb.auug.org.au>
MIME-Version: 1.0
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  Documentation/virt/kvm/api.rst

between commit:

  b318e8decf6b ("KVM: x86: Protect userspace MSR filter with SRCU, and set atomically-ish")

from Linus tree and commit:

  24e7475f931a ("doc/virt/kvm: move KVM_CAP_PPC_MULTITCE in section 8")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc Documentation/virt/kvm/api.rst
index 245d80581f15,fd4a84911355..000000000000
--- a/Documentation/virt/kvm/api.rst
